index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="container">
+        <main>
+            <p class="paragraph_one">About <br>Myself</p>
+            <ul class="paragraph">
+                <li class="list_item_one">Hello I’m a software developer!
+                    I can help you build a product ,
+                    feature or website Look through some of
+                    my work and experience! If you like what
+                    you see and have a project you need coded,
+                    don’t hestiate to contact me.
+                </li>
+                <li class="list_item_two">LET'S CONNECT</li>
+            </ul>
+            <div class="social-media-icons">
+                <ul class="icons">
+                    <li><img src="images/twitter.png" alt="twitter"></li>
+                    <li><img src="images/linkedin.png" alt="linked in"></li>
+                    <li><img src="images/mid.png" alt="mid"></li>
+                    <li><img src="images/git.svg" alt="git"></li>
+                    <li><img src="images/fin.svg" alt="fing"></li>
+                </ul>
+            </div>
+            <button type="button">Get my resume</button>
+
+            <p class="section1"> Languages <img src="images/Union.png" alt="drop down" /></p>
+            <ul class="languages">
+                <li class="l-items"> <img src="images/js.png" alt="java">Javascript</li>
+                <li class="l-items"> <img src="images/html.png" alt="html">HTML</li>
+                <li class="l-items"> <img src="images/CSS.png" alt="css">CSS</li>
```

Comment:
- [ ] According to your Figma template, please make sure that the text and the icons are properly centered, if you are using flex-box this could be achieved easily by adding `align-items: center`

Your project:
![image](https://user-images.githubusercontent.com/52635378/165316917-ccb2b8f3-d8f2-4979-a021-1d79f99a6677.png) 

Figma template:
 ![image](https://user-images.githubusercontent.com/52635378/165317226-c6e86367-16a4-4078-8e71-db57de3a211b.png)